common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/listitem/OneLineListItem.kt

```diff
@@ -90,6 +92,11 @@ class OneLineListItem @JvmOverloads constructor(
         binding.primaryText.text = title
     }
 
+    /** Sets title text color */
+    fun setPrimaryTextColor(@ColorRes colorRes: Int) {
```

Comment:
this should be a ColorStateList and not just a color. Otherwise we open the API for cases that don't handle enabled / disabled.

Reply:
True! I will change it